### PR TITLE
SLVS-858 S3600 - "params" should not be introduced on overrides

### PR DESCRIPTION
**Pull request by tvsonar**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 version: 1.0.{build}
-os: Visual Studio 2015
+os: Previous Visual Studio 2015
```
Comment (tvsonar) [inline appveyor.yml]: this should be removed

File: src/SonarLint.CSharp/Rules/MethodOverrideAddsParams.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            new DiagnosticDescriptor(DiagnosticId, Title, MessageFormat, Category,
+                RuleSeverity.ToDiagnosticSeverity(), IsActivatedByDefault,
+                helpLinkUri: DiagnosticId.GetHelpLink(),
+                description: Description);
+
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get { return ImmutableArray.Create(Rule); } }
+
+        protected override void Initialize(SonarAnalysisContext context)
+        {
+            context.RegisterSyntaxNodeActionInNonGenerated(
```
Comment (dbolkensteyn) [inline src/SonarLint.CSharp/Rules/MethodOverrideAddsParams.cs]: why not code block action which provides both the syntax and symbol from its context?

**Comment by dbolkensteyn** [comment]
LGTM

File: src/SonarLint.CSharp/Rules/MethodOverrideAddsParams.cs (hunk 2 above)
Comment (tvsonar) [inline src/SonarLint.CSharp/Rules/MethodOverrideAddsParams.cs]: same reason as on the other PR, we don't want to get called back on properties.

**Pull request merged**